reconcile/utils/unleash.py

```diff
@@ -51,6 +51,20 @@ def apply(self, context: Optional[dict] = None) -> bool:
         return enable
 
 
+class EnableClusterStrategy(Strategy):
+    def load_provisioning(self) -> list:
```

Comment:
Could create a ClusterStrategy class, that implements this only once.

Reply:
done. PTAL